content/md/en/docs/reference/command-line-tools/srtool.md

```diff
@@ -1,9 +1,416 @@
 ---
 title: srtool
-description: Command-line reference information for the srtool program.
+description: Describes the Docker container and command-line interface for building the Substrate WebAssembly runtime.
 keywords:
 ---
 
-The `srtool` allows building WASM runtimes in a deterministic way, allowing CIs and users to produce a strictly identical WASM runtime.
+The core component of the Substrate runtime toolbox (`srtool`) is a Docker container that enables you to build the Substrate WebAssembly runtime in a deterministic way.
+By using this tool, you can ensure that the same source code consistently reproduces an identical WebAssembly blob. 
+You can also use the tool to inspect and audit the runtime for any Substrate-based chain and to integrate building the WebAssembly runtime into your CI/CD pipeline.
 
-- [Go to documentation](https://github.com/paritytech/srtool).
+## Working with the Docker container
+
+Because `srtool` is a Docker container, you must have Docker available in your build environment to use it.
+However, you don't need to know anything about using Docker to build a chain using `srtool`.
+Instead, you can use the following programs to interact with the Docker image.
+
+- [srtool-cli](https://github.com/chevdor/srtool-cli) provides a command-line interface to pull the `srtool` Docker image and build the runtime using the `srtool` Docker container.
+
+- [subwasm](https://github.com/chevdor/subwasm) provides command-line options for working with the metadata and WebAssembly runtime built using `srtool`.
+
+- [srtool-actions](https://github.com/chevdor/srtool-actions) provides Github actions to integrate builds produced using the `srtool` image with your CI/CD pipeline.
+  
+- [srtool-app](https://gitlab.com/chevdor/srtool-app) provides a simple graphical user interface for building the runtime using the `srtool` Docker image.
+
+## Image naming convention
```

Comment:
This section could move  all the way down below the `srtool-cli`.
Also, people enabling the `srtool-cli` logger can easily see and learn about what happens under the hood.
That makes the info about Docker image naming not so relevant.

FYI: Both `srtool-cli` and `srtool-actions` have been primary made so the normal user does NOT need to know what is the canonical version of the srtiool image at the moment, those tools know how to check and fetch **the right** image/version.

Reply:
I've decided to split the information into two separate topics, one that has the why and how and this one focused on the command usage and examples.